<launch>

    <!-- <node pkg="swipe_obstacles" type="swipe_detector_fixed_node" name="detector_node"/> -->
    <node pkg="swipe_obstacles" type="swipe_position_manager_node" name="manager_node"/>
    <node pkg="swipe_obstacles" type="swipe_visualizer_node" name="visualizer_node"/>
    <node pkg="swipe_obstacles" type="swipe_reflector_node" name="reflector_node"/>

</launch>
